Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            string decision;
+            do
+            {
+                Console.Clear();
+                decision = AddNewPerson();
+
+            } while (decision == "Y");
+        }
+
+        public static string AddNewPerson()
+        {
+            string name = PromptNameString("What is your name?");
+            float age = PromptInt("How old are you?");
+            float weight = PromptFloat("What is your weight (in kg)?");
+            float height = PromptFloat("What is your height (in cm)?");
+
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
+
+            float heightInMeters = height / 100;
+            float bmi = CalculateBmi(weight, heightInMeters);
+            Console.WriteLine($"His/her BMI = {bmi:N}");
+
+            Console.WriteLine("Do you want to add another person (Y/y = Yes, everything else = No)?");
+            string decision = Console.ReadLine();
+            string decisionToUpper = decision.ToUpperInvariant();
+
+            return decisionToUpper;
+        }
+
+        public static string PromptNameString(string message)
+        {
+            Console.WriteLine(message);
+
+            string name = Console.ReadLine();
+
+            if (String.IsNullOrWhiteSpace(name))
+            {
+                Console.Write("Name cannot be empty.");
+                return "-";
+            }
+            else
+            {
+                return name;
+            }
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            bool isValidHeightAndWeight = height > 0 && weight > 0;
+            string heightError = "Height cannot be equal or less than zero, but was ";
+            string heightErrorForNegative = "Height cannot be less than zero, but was ";
+            string weightError = "Weight cannot be equal or less than zero, but was ";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'weightError'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4971302)